<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultShow</class>
 <widget class="QMainWindow" name="ResultShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	font: normal normal 14px \&quot;STXingkai\&quot;;
}</string>
    </property>
    <property name="text">
     <string>## 出发点:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="startPoint">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>## 终点:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="endPoint">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="searchWay">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>最快时间</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>最短距离</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>最少换乘数</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label1_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>## 搜索方案</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">label-&gt;setStyleSheet(&quot;background:transparent&quot;)</string>
    </property>
    <property name="text">
     <string>## 时间：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">label-&gt;setStyleSheet(&quot;background:transparent&quot;)</string>
    </property>
    <property name="text">
     <string>## 距离：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>## 换乘数：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>## 总站数：</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>## 乘坐方案</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="distance">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exchange">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>330</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="numOfSubway">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label1_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>200</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>## 分钟</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1_10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>## 千米</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1_11">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>## 次</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label1_12">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>## 站</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>271</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Path">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>textBrowser_3</zorder>
   <zorder>textBrowser_2</zorder>
   <zorder>textBrowser</zorder>
   <zorder>label1</zorder>
   <zorder>startPoint</zorder>
   <zorder>label1_2</zorder>
   <zorder>endPoint</zorder>
   <zorder>searchWay</zorder>
   <zorder>label1_3</zorder>
   <zorder>label1_4</zorder>
   <zorder>label1_5</zorder>
   <zorder>label1_6</zorder>
   <zorder>label1_7</zorder>
   <zorder>label1_8</zorder>
   <zorder>time</zorder>
   <zorder>distance</zorder>
   <zorder>exchange</zorder>
   <zorder>numOfSubway</zorder>
   <zorder>label1_9</zorder>
   <zorder>label1_10</zorder>
   <zorder>label1_11</zorder>
   <zorder>label1_12</zorder>
   <zorder>Path</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
